### PR TITLE
Notebook fix

### PR DESCRIPTION
**Pull request by Corey-Zumar**

### CODE REVIEW

**Comment by Corey-Zumar** [comment]
Here's an additional portion of the description that could use some editing / polishing:

Under "Deploy to Clipper" - this is a long sentence, hard to parse
```
"Clipper provides a collection of model deployer modules to simplify the process of deploying a trained model to Clipper and avoid the need to figure out how to save models and build custom Docker containers capable of serving the saved models for some common use cases"
```

**Comment by dcrankshaw** [comment]
Thanks for the feedback. I incorporated the changes directly (merging ipython notebooks with git is messy).
